### PR TITLE
E2e

### PR DESCRIPTION
**Pull request by lauradobson**
Updated:

1. E2E testing instructions
2. Documentation
3. Changed "Export Results" name on home page

### CODE REVIEW

**Review by amyleblond — APPROVED** [review]

File: Documentation/README.md [hunk 1]
```diff
@@ -36,7 +36,20 @@
 		* Once your file is approved you will be able to see a preview of the data you are submitting to the database.
 		* Verify one last time that the information is correct. Make changes if required.
 		* Submit the course information to the database.
-2. [Change Application Form]
+2. [Open/Close the TA Application]
```
Comment (smwatts) [inline Documentation/README.md:39]: I'd add a comment about how the application form as an intro & the actual application, which shows just the application (I know that was a point of confusion for Smucker)

File: Documentation/README.md [hunk 2]
```diff
@@ -36,7 +36,20 @@
...
+		** 1 - The AC who wants to change the status of the application. 
+		** 2 - The AC wants to view the content of the form, including the front matter if it gets updated from the "Change Application Form" step.
+		** 3 - Students when they want to submit an application. It is important to note that they will NOT have access to the Open/Close funtionality from Scenario 1.
+	* How to Use If You Are the AC: 
+		* Refer to Step 0
+		* Open Step 3, Open/Close the TA Application - (https://team4.uwaterloo.ca/taform/application.html)
+		* Press "Change Status" to change the status of the application. 
+		* When the first line "Application Status" is red and says "Closed", then the form is not live and students can not apply or access the application page.
+		* When the first line "Application Status" is green and says "Open", then the form is live and students can apply.
+		* You are able to view the from matter of the form. The front matter consists of everything below the Application Status functionality to the top of "Basic Information".
```
Comment (smwatts) [inline Documentation/README.md]: From matter -> front matter

File: Documentation/README.md [hunk 3]
```diff
@@ -36,7 +36,20 @@
 		* Once your file is approved you will be able to see a preview of the data you are submitting to the database.
 		* Verify one last time that the information is correct. Make changes if required.
 		* Submit the course information to the database.
-2. [Change Application Form]
+2. [Open/Close the TA Application]
+	* Function: This page allows the AC view the application page which is what the student's see. Additionally, this page allows the AC to change the status of the application page between "Closed" and "Open". 
+	* Scenarios for Use: 
+		** 1 - The AC who wants to change the status of the application. 
+		** 2 - The AC wants to view the content of the form, including the front matter if it gets updated from the "Change Application Form" step.
+		** 3 - Students when they want to submit an application. It is important to note that they will NOT have access to the Open/Close funtionality from Scenario 1.
```
Comment (smwatts) [inline Documentation/README.md]: * unless they logout of the system

File: Documentation/README.md [hunk 4]
```diff
@@ -36,7 +36,20 @@
...
+	* How to Use If You Are the AC: 
+		* Refer to Step 0
+		* Open Step 3, Open/Close the TA Application - (https://team4.uwaterloo.ca/taform/application.html)
+		* Press "Change Status" to change the status of the application. 
+		* When the first line "Application Status" is red and says "Closed", then the form is not live and students can not apply or access the application page.
```
Comment (smwatts) [inline Documentation/README.md]: Students can access the page, they just get a message saying "the application is closed at this time". Maybe change the wording?

File: Documentation/README.md [hunk 5]
```diff
@@ -104,7 +117,15 @@ Last step. [Go on the algorithm page](https://team4.uwaterloo.ca/taform/algorith
 	* Unhappy with results?
 		* Exclude students, adjust student ratings, adjust instructor ratings or adjust number of positions available and run it again.
 		* Another alternative is exporting the course_info and student_info from the previous step and run it with the old excel heuristic.
-6. [Send Ranking Links, Monitor Replies]
+7. [Assign Number of TAs]
+	* Function: This page allows the AC to update the number of TAs needed per course. The AC can enter in integer values (including 0) for the following type of TA positions:
```
Comment (smwatts) [inline Documentation/README.md:122]: I'd remove (including 0) because this might make it confusing.

**Review by smwatts — COMMENTED** [review]
Please change wording

**Review by smwatts — APPROVED** [review]
LGTM

**Pull request merged**